strings.txt

```diff
@@ -17810,3 +17810,601 @@
     sk = od %d
     es = desde %d
     pl = od %d
+
+  [meter]
```

Comment:
См выше
> On Apr 26, 2016, at 4:35 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61085623>:
> 
> > @@ -17810,3 +17810,601 @@
> >      sk = od %d
> >      es = desde %d
> >      pl = od %d
> > +
> > +  [meter]
> По аналогии с существующим переводом лучше назвать [metres]
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/454ca48abdc1a7a643a178fac869c249f100121f..9460d6ca1e36b2c17bee62d5d712319e41eb236f#r61085623>